test/facade_converter_test.ts

```diff
@@ -107,6 +123,13 @@ ByteBuffer buff;`);
 
 var y = x is Promise;`);
     });
+
+    it('works with a different typings directory', () => {
+      expectWithTypingsRootTypes('var y: Promise = Promise.resolve(1);')
```

Comment:
No it's the same logic you have now, I'm just saying that if you add another Promise symbol in another place, and set the prefix to point to that place, you can keep this same assertion with less duplicate fixture. Does that make sense?